frontend/src/components/Status.vue

```diff
@@ -1,16 +1,24 @@
 <template>
-  <span :class="{ 'is-online': true, online: isOnline }"/>
+  <span
+    class="is-online"
+    :class="{ online: isOnline, hidden: isLogin }"/>
 </template>
 
 <script>
 import { mapState } from 'vuex';
 import { IS_ONLINE } from '../store';
+import { LOGIN_PATH } from '../router/constants';
 
 export default {
   name: 'Refresh',
   computed: {
     ...mapState([IS_ONLINE]),
   },
+  data() {
+    return {
+      isLogin: this.$router.currentRoute.path === LOGIN_PATH,
```

Comment:
Is there any way in Vue to extract setting this property and share it between components? It is repeated a couple of times.

Reply:
I can see where you are coming from, thought I am trying to get away from monolithic architecture with the main view class holding entire knowledge for the entire app. Thus it is not bound to the `vue` object. 
I could extract it to the helper function, but it would just compare two strings, so I decided to leave it as it is